ZIGSIMPlus/Presenters/CommandDataSettingPresenter.swift

```diff
@@ -0,0 +1,185 @@
+//
+//  CommandDataSettingPresenter.swift
+//  ZIGSIMPlus
+//
+//  Created by YoneyamaShunpei on 2019/05/28.
+//  Copyright © 2019 Nozomu Kuwae. All rights reserved.
+//
+
+import Foundation
+import SwiftyUserDefaults
+
+protocol CommandDataSettingPresenterProtocol {
+    func initUserDefault()
+    func getUserDefaultTexts() -> Dictionary<String,String>
+    func getUserDefaultSegments() -> Dictionary<String,Int>
+    func updateTextsUserDefault(texts: Dictionary<String, String>)
+    func updateSegmentsUserDefault(segmentControls: Dictionary<String, Int>)
+}
+
+protocol CommandDataSettingPresenterDelegate: AnyObject {}
+
+final class CommandDataSettingPresenter: CommandDataSettingPresenterProtocol {
+    var view: CommandDataSettingPresenterDelegate!
+    func initUserDefault(){
+        AppSettingModel.shared.address = Defaults[.userIpAdress]?.description ?? ""
+        AppSettingModel.shared.port = Int32(Defaults[.userPortNumber]?.description ?? "") ?? 0
+        AppSettingModel.shared.deviceUUID = Defaults[.userDeviceUUID]?.description ?? ""
+        AppSettingModel.shared.beaconUUID = Defaults[.userBeaconUUID]?.description ?? ""
+
+        if Defaults[.userDataDestination] == 0 {
```

Comment:
There are many `if` statements, and some of them can be avoided by accessing enum rawValue.